modules/cnf-topology-aware-lifecycle-manager-PAO-update.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+// Epic CNF-4767 (4.11), Story TELCODOCS-593
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talm-PAO-update_{context}"]
+= Removing Performance Addon Operator subscriptions from deployed clusters
+
+In earlier versions of {product-title}, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator. 
+
+Do not install the Performance Addon Operator on clusters running {product-title} 4.11 or later. When you upgrade to {product-title} 4.11, the Node Tuning Operator automatically removes the Performance Addon Operator. However, you  need to manually remove any Performance Addon Operator subscriptions to prevent a reinstallation of the Operator. The reference DU profile includes the Performance Addon Operator in the `common-ranGen.yaml` `PolicyGenTemplate`. To remove the subscription from deployed spoke clusters, you must update `common-ranGen.yaml`.
+
```

Comment:
Maybe add a note here that the latest 4.10 version of the PAO will detect when it is running on a 4.11 cluster and automatically hibernate so as to not interfere with the NTO functionality. However it is still recommended to fully remove the operator from 4.11 OCP clusters.

Reply:
Is there a particular version of PAO we should mention? For example, PAO version 4.10.4 or later

Reply:
This change is included in 4.10.3-5 or later. Thank you!